login-workflow/src/screens/SiteOptionsScreen/SiteOptionsScreenBase.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { SiteOptionsScreenProps } from './types';
+import { WorkflowCard, WorkflowCardHeader, WorkflowCardBody, WorkflowCardInstructions } from '../../components';
+import Box from '@mui/material/Box';
+import CardActions from '@mui/material/CardActions';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+/**
+ * Component that renders a success screen
+ *
+ * @param icon the icon to be displayed on the screen
+ * @param message success message to be displayed on the screen
+ * @param joinExistingOrgLabel to display label for the custom button which redirects to existing organization flow
+ * @param createNewOrgLabel display label for the custom button which redirects to create a new organization flow
+ * @param onJoinExistingOrg function to call when user wants to join existing organization
+ * @param onCreateNewOrg function to call when user wants to create a new  organization
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ *
```

Comment:
you can add @param errorDisplayConfig configuration for customizing how errors are displayed to the list

Reply:
only because I see it in your types but not being used here...see next comment

Reply:
added `@param errorDisplayConfig`